common.php

```diff
@@ -338,6 +349,45 @@
         'exclude_admin_structure_param_list' => array(
             'Worklist',
         ),
+      // Settings for OneLogin PHP-SAML toolkit
+      'SAML_settings' => array(
+        // BaseURL of the view that will process the SAML message
+        'baseurl' => '',
```

Comment:
Spacing needs to be corrected. It seems like its 2, needs to be changed to 4.

Make sure your IDE is configured correctly.